modules/dataframeqt/include/inviwo/dataframeqt/dataframetableprocessorwidget.h

```diff
@@ -48,10 +47,7 @@ class DataFrameTableView;
  * \brief A processor widget showing a DataFrame in a table view.
  */
 class IVW_MODULE_DATAFRAMEQT_API DataFrameTableProcessorWidget : public ProcessorWidgetQt {
-#include <warn/push>
```

Comment:
no longer needed?

Reply:
I have removed them in most places and not seen any warnings...